src/components/Profile/Profile.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import s from './Profile.module.css';
+import MyPosts from './MyPosts/MyPosts';
+
+const Profile = () => {
+    return (
+        <div className='content'>
+            <div>
+                <img src='https://a.d-cd.net/870b45as-960.jpg'></img>
+            </div>
+            <div>
+                ava+desc
```

Comment:
Consider storing "ava+desc" in a separate constant, preferably an object - if you have multiple strings like this in your app.

Reply:
This is a workpiece, of course, it will be stored in the object with subsequent development